djangocms_version_locking/monkeypatch/cms_toolbars.py

```diff
@@ -20,18 +40,29 @@ def inner(self, **kwargs):
             func(self, **kwargs)
             return
 
+        # Populate a title with the locked author details
+        title = ""
+        version_lock = get_lock_for_content(self.toolbar.obj)
+        if version_lock:
+            # If the users name is available use it, otherwise use their username
+            title = _("Locked with {name}").format(
+                name=version_lock.created_by.get_full_name() or version_lock.created_by.username,
+            )
+
         # There is a version lock for the current object.
         # Add a disabled edit button.
-        item = ButtonList(side=self.toolbar.RIGHT)
-        item.add_button(
+        item_list = ButtonList(side=self.toolbar.RIGHT)
+        button = ButtonWithAttributes(
             format_html(
                 '<span style="vertical-align:middle;position:relative;top:-1px" class="cms-icon cms-icon-lock"></span>{name}',
                 name=_('Edit'),
             ),
             url='javascript:void(0)',
             disabled=True,
             extra_classes=['cms-btn-action', 'cms-version-locking-btn-icon'],
+            html_attributes={'title': title},
```

Comment:
I think you should only set title attribute when needed (currently it will always be set, even if empty)